authmanager/authmanager.go

```diff
@@ -19,21 +20,31 @@ const (
 	pivotalAPITokenMsg = "Please go to https://www.pivotaltracker.com/profile, create a new API token & paste it here:\nToken: "
 	redmineAPITokenMsg = "Please go to %s/my/account, create a new API token & paste it here:\nToken: "
 
+	githubAPITokenWarning = "WARNING: Github has API rate limits for all requests which do not contain a token.\nPlease create a read-only access token to increase that limit. https://github.com/settings/tokens"
```

Comment:
Could you instead link https://developer.github.com/v3/#rate-limiting to this warning message.

Reply:
This link shows details about why one might want to consider adding an apitoken